frontend/src/services/vdyp-api/apis/run-dcsvprojection-api.ts

```diff
@@ -0,0 +1,152 @@
+/* tslint:disable */
```

Comment:
We shouldn't be disabling tslint. If you're getting linting errors, it's better to correct them vs. disabling the checks.

Note that Sonar will check them anyway and they'll need to be corrected regardless.